tests/integration_tests_control_files/advection_explicit_periodic/2D_advection_explicit_periodic_energy.prm

```diff
@@ -0,0 +1,52 @@
+# -------------------
+
+set test_type = advection_periodicity
+
+# Number of dimensions
+set dimension = 2
+
+set use_weak_form = false
+
+set overintegration = 0
+
+set use_collocated_nodes = false
+
+set use_split_form = false
+
+set rk_order = 4
+
+set use_curvilinear_split_form = true
+
+set use_weight_adjusted_mass = true
+
+set use_periodic_bc = true
+
+set use_energy = true
+
+set flux_reconstruction = cPlus
+
+set use_classical_FR = false
+
+# The PDE we want to solve
+set pde_type = advection
+
+#set conv_num_flux = lax_friedrichs
+set conv_num_flux = central_flux
+
```

Comment:
We are getting to the point where "AllParameters" includes too many parameters of its own.

Create subsection for SpatialDiscretization to group up the relevant parameters.

Reply:
will be fixed by issue #124